extensions.json

```diff
@@ -249,6 +249,11 @@
       "repository": "https://github.com/primer/github-vscode-theme",
       "checkout": "v1.1.2"
     },
+    {
+      "id": "golang.Go",
+      "repository": "https://github.com/golang/vscode-go",
+      "checkout": "latest"
```

Comment:
One more thing, is `latest` always the best branch to publish from? (I notice there is a `release` branch as well)

Reply:
Yes, we plan to use the `release` branch to be the branch for release preps (rc, beta, stable), and the `latest` branch to be always pointing to the stable version.